### DISCUSSION TITLE
windows compatibility?

### DISCUSSION THREAD

**Discussion by killertuxX** [notro/gud#16] [Q&A]
Is there any way to make it work with a windows machine? on fedora it works simply by plugging it in and the machine recognizes the pi, but on windows i only see a "raspberry pi zero display gadget" device and it says that the driver is unavailable

**Comment by notro** [comment]
That would require someone to write a driver for Windows. I have no experience writing Windows drivers so it's not on top of my TODO for gud. I did a search a while back to see if there was any example code out there and found this: https://www.apriorit.com/dev-blog/605-develop-driver-graphics-adapter-example

**Comment by hadi77ir** [comment]
I think [QXL DOD driver](https://github.com/freedesktop/spice-qxl-wddm-dod) is a good starting point for everyone who wants to contribute.
